src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -395,16 +395,16 @@ tests:
           [1708071897.294574][7731:7733] CHIP:TOO:         ClusterID: 6
           [1708071897.294580][7731:7733] CHIP:TOO:         AttributeValueList: 1 entries
           [1708071897.294589][7731:7733] CHIP:TOO:           [1]: {
-          [1708071897.294593][7731:7733] CHIP:TOO:             AttributeID: 1
-          [1708071897.294596][7731:7733] CHIP:TOO:             AttributeValue: 1
+          [1708071897.294593][7731:7733] CHIP:TOO:             AttributeID: 0
```

Comment:
was this a error before or why does the attribute id changes?

Reply:
Yes, there was an error in the comment, it should have been attribute ID 0 from the start.